engine/graphics/src/test/test_graphics.cpp

```diff
@@ -484,6 +484,32 @@ TEST_F(dmGraphicsTest, TestProgram)
     dmGraphics::DeleteFragmentProgram(fp);
 }
 
+TEST_F(dmGraphicsTest, TestComputeProgram)
+{
+    const char* compute_data = ""
+        "#version 430 core\n"
+        "layout(local_size_x = 1, local_size_y = 1, local_size_z = 1) in;\n"
```

Comment:
Later on we should probably parse this string to get the work units, but it's not needed right now.